### ISSUE TITLE
use model.eval() to deal with dropout during evaluation phase

### ISSUE THREAD

**Issue by raphaelreinauer** [giotto-ai/giotto-deep#40]
To compute the model accuracy correctly for dropout use model.eval(), see
https://stackoverflow.com/questions/53879727/pytorch-how-to-deactivate-dropout-in-evaluation-mode
in the code
https://github.com/giotto-ai/giotto-deep/blob/2965f49a5653bc51790bc76c66e21f2e5ca3e65d/gdeep/pipeline/pipeline.py#L94
